kuzminskiy_andrey/gen_graph.cpp

```diff
@@ -0,0 +1,88 @@
+// uncomment the 85nd line to output the graph
+
+#include <iostream>
+#include <vector>
+
+class Graph {
+  using VertexId = int;
+  using EdgeId = int;
+
+ public:
+  void add_vertex() { kVerticesCount++; }
+
+  void add_edge(VertexId from_vertex_id, VertexId to_vertex_id) {
+    edges.emplace(edges.end(),
+                  (Edge(kEdgesCount++, from_vertex_id, to_vertex_id)));
```

Comment:
Похоже что у тебя тут баг.
Добавь приватный метод 
```cpp
bool has_vertex(VertexId id) const;
```
И добавь внутрь текущего метода проверки на то что вершины существуют:
```suggestion
  void add_edge(VertexId from_vertex_id, VertexId to_vertex_id) {
    assert(has_vertex(from_vertex_id));
    assert(has_vertex(to_vertex_id));
    edges_.emplace_back(...);
```